ui/src/main/webapp/src/app/shared/validators/project-name-not-exists.validator.ts

```diff
@@ -21,6 +24,10 @@ export class ProjectNameNotExistsValidator implements Validator {
         let projectService = this.projectService;
 
         return new Promise(resolve => {
+            if (this.editMode) {
+                resolve(null);
```

Comment:
This is only bypassing the name duplication check. I think the rest of the validation still happens.

I don't know of a better way to bypass only one validator when in edit mode.